docs/core/diagnostics/built-in-metrics-efcore.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Entity Framework Core metrics
+description: Review the metrics available for Entity Framework Core
+ms.topic: reference
+ms.date: 10/06/2024
+---
+
+# Entity Framework Core metrics
+
+This article describes the metrics built-in for Entity Framework Core produced using the <xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API.
+
+## `Microsoft.EntityFrameworkCore`
+
+The `Microsoft.EntityFrameworkCore` metrics report information about operations done by Entity Framework Core:
+
+- [`microsoft.entityframeworkcore.active_dbcontexts`](#metric-microsoftentityframeworkcoreactive_dbcontexts)
+- [`microsoft.entityframeworkcore.queries`](#metric-microsoftentityframeworkcorequeries)
+- [`microsoft.entityframeworkcore.savechanges`](#metric-microsoftentityframeworkcoresavechanges)
+- [`microsoft.entityframeworkcore.compiled_query_cache_hit_rate`](#metric-)
+- [`microsoft.entityframeworkcore.execution_strategy_operation_failures`](#metric-microsoftentityframeworkcoreexecution_strategy_operation_failures)
+- [`microsoft.entityframeworkcore.optimistic_concurrency_failures`](#metric-microsoftentityframeworkcoreoptimistic_concurrency_failures)
+
+##### Metric: `microsoft.entityframeworkcore.active_dbcontexts`
+
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `microsoft.entityframeworkcore.active_dbcontexts` | ObservableUpDownCounter | `{dbcontext}` | Number of currently active `DbContext` instances. |
+
+Available starting in: .NET 9.0.
```

Comment:
```suggestion
Available starting in: EF 9.0.
```

BTW we could just mention this once at the top (we can always change in the future if we add new metrics etc.)

Reply:
I'm following what other pages use. And i.e. ASP.NET Core uses ".NET" and has it under every counter. Makes sense to me to keep it consistent and have it same even if we add more in the future.

Reply:
ASP.NET is part of the shared network, so there's a link between ASP.NET and .NET which EF (a pure nuget package) doesn't have... For example, it's possibly to use an older version of EF with .NET 9.0, at which point these counters don't exist.

Reply:
Fine with me.